modular_skyrat/modules/bluespace_miner/code/bluespace_miner.dm

```diff
@@ -0,0 +1,167 @@
+/obj/machinery/bluespace_miner
+	name = "bluespace miner"
+	desc = "Through the power of bluespace, it is capable of producing materials."
+	icon = 'modular_skyrat/modules/bluespace_miner/icons/bluespace_miner.dmi'
+	icon_state = "miner"
+
+	density = TRUE
+	circuit = /obj/item/circuitboard/machine/bluespace_miner
+	idle_power_usage = 200
+
+	///the temperature of the co2 produced per successful process (its really 100)
```

Comment:
What unit are you using? Kelvins or Celcius?

Reply:
its kelvin